api/src/data/index.js

```diff
@@ -1,31 +1,29 @@
 // Imports
 import fixtures from 'sequelize-fixtures';
 import process from 'process';
+import generateInvoices from './invoice/generateInvoices';
+import generateBlocks from './block/generateBlocks';
 
 export default function (models) {
   console.log('SETUP - Starting to populate tables with Initial Data');
 
-  return new Promise((resolve, reject) => {
-    fixtures.loadFile(`${__dirname}/estado/estados.js`, models, { log: () => {} })
-      .then(() => {
-        fixtures.loadFile(`${__dirname}/regiao/regioes.js`, models, { log: () => {} })
-          .then(() => {
-            // fixtures.loadFile(`${__dirname}/municipio/municipios.js`, models, { log: () => {} })
-            // .then(() => {
-            fixtures.loadFile(`${__dirname}/user/users.js`, models, { log: () => {} })
-              .then(() => {
-                fixtures.loadFile(`${__dirname}/client/clients.js`, models, { log: () => {} })
-                  .then(() => {
-                    fixtures.loadFile(`${__dirname}/authorizationCode/authorizationCodes.js`, models, { log: () => {} })
-                      .then(() => {
-                        process.emit('dataLoaded');
-                        resolve();
-                      });
-                  });
-              });
-            // });
-          });
-      })
-      .catch(err => reject(err));
+  return new Promise(async (resolve, reject) => {
+    try {
+      await fixtures.loadFile(`${__dirname}/estado/estados.js`, models, { log: () => {} });
+      await fixtures.loadFile(`${__dirname}/regiao/regioes.js`, models, { log: () => {} });
+      // fixtures.loadFile(`${__dirname}/municipio/municipios.js`, models, { log: () => {} })
```

Comment:
Sugestão:

```
if (process.env.NODE_ENV === 'production') {
  await fixtures.loadFile(`${__dirname}/municipio/municipios.js`, models, { log: () => {} })
} else {
  await fixtures.loadFile(`${__dirname}/municipio/municipiosSmall.js`, models, { log: () => {} });
}
```

Reply:
Feito. Coloquei `await generateInvoices(500); await generateBlocks(4);` pra rodar também somente se *não* for production.